core/domain/summary_services.py

```diff
@@ -382,37 +377,35 @@ def get_displayable_exp_summary_dicts(exploration_summaries):
     displayable_exp_summaries = []
 
     for ind, exploration_summary in enumerate(exploration_summaries):
-        if not exploration_summary:
```

Comment:
Why is this changed? Won't both do the same thing?

Reply:
Yes I changed it so it becomes easy to test